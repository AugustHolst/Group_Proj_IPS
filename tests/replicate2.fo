fun bool writeBool(bool b) = write(b)

fun int writeInt(int b) = write(b)

fun [bool] main() =
	let ts = replicate(2, true) in
	let ints = replicate(2, 5) in
	let fs = replicate(2, false) in
	let a = map(writeInt, ints) in
	let b = map(writeBool, fs) in
	map(writeBool, ts)