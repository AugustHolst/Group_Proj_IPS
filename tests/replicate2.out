5 5 falsefalsetruetrue
